dmn/src/main/java/io/camunda/zeebe/dmn/EvaluatedDecision.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.dmn;
+
+import java.util.List;
+import org.agrona.DirectBuffer;
+
+/**
+ * An evaluated DMN decision. It contains details of the evaluation depending on the decision type.
+ */
+public interface EvaluatedDecision {
```

Comment:
🔧 I personally like it when the javadoc is navigatable in the tooltips provided by intellij. Please link this interface in the java doc of `DecisionResult` (something like, that it also contains the details of making the decision) and link back to it from here. This is not mandatory, just a suggestion.